### PR TITLE
DOCS-9983: FTPS Connector M4. Improve docs regarding TLS Session Reuse

### PR DESCRIPTION
**Pull request by giss-ignacio**

### CODE REVIEW

**Review by nicomz — DISMISSED** [review]

**Comment by giss-ignacio** [comment]
deleted branch by mistake, reopening PR. 
Sorry

File: ftps/1.6/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -128,6 +128,29 @@ You can use Transport Layer Security (TLS) and configure FTPS by providing a key
 * For more on SSL configuration in Mule, see xref:mule-runtime::tls-configuration.adoc[About TLS Configuration].
 * For more on other functionality provided by this connector, see xref:ftp-connector::index.adoc[FTP Connector Documentation Reference].
 
+== Configure Java JDK for Session Reuse
+
+To reuse sessions using FTPS you must disable the Extended Master Secret extension for compatibility (JDK-8192045), which by default is set to true. The extension protects users from vulnerabilities by using the same session on different groups of clients and servers, which contradicts the demand to use the same session for control and data connections. Extended Master Secret extension activated, enables session reuse only with an endpoint verification, which is implemented for HTTPS and LDAP only.
```
Comment (kbostrom) [inline ftps/1.6/modules/ROOT/pages/index.adoc]:
```suggestion
To reuse sessions using FTPS you must disable the Extended Master Secret extension for compatibility (JDK-8192045), which by default is set to `true`. The extension protects users from vulnerabilities by using the same session on different groups of clients and servers, which contradicts the demand to use the same session for control and data connections. With the Extended Master Secret extension activated, session reuse is possible only by endpoint verification, which is implemented for HTTPS and LDAP only.
```
However...why not keep the first sentence and omit all the rest? The bulk of this intro. is about the default, activated state of the extension, not the desired, disabled state.

File: ftps/1.6/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -128,6 +128,29 @@ You can use Transport Layer Security (TLS) and configure FTPS by providing a key
...
+
+To disable the Extended Master Secret extension:
+
+* If you deploy your Mule application to a Standalone Mule runtime engine, add the following argument to the JVM execution:
```
Comment (kbostrom) [inline ftps/1.6/modules/ROOT/pages/index.adoc]:
```suggestion
* If you deploy your Mule application to a standalone Mule runtime engine, add the following argument to the JVM execution:
```

File: ftps/1.6/modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -128,6 +128,29 @@ You can use Transport Layer Security (TLS) and configure FTPS by providing a key
...
++
+`./bin/mule -Djdk.tls.useExtendedMasterSecret=false`
+
+* If you run your Mule application in Anypoint Studio, follow the next steps:
```
Comment (kbostrom) [inline ftps/1.6/modules/ROOT/pages/index.adoc]: Why "run" here but "deploy" above?

Comment (kbostrom) [reply ftps/1.6/modules/ROOT/pages/index.adoc]: the next > these

File: ftps/1.6/modules/ROOT/pages/index.adoc (hunk 2 above)
Comment (kbostrom) [reply ftps/1.6/modules/ROOT/pages/index.adoc]: Another way to present these two scenarios is as follows:

"To disable the Extended Master Secret extension when working in a standalone Mule environment, add the following...:

"To disable the extension when working in Anypoint Studio, follow these steps:

**Review by kbostrom — DISMISSED** [review]
CE

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**